src/client/containers/RegistrationPage/StudentRegistrationPage/StudentRegistrationPage.styles.css

```diff
@@ -0,0 +1,39 @@
+.student-registration-main {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  flex-direction: column;
+}
+
+.student-registration-avatar {
+  width: 100%;
+  margin: 0;
+  padding-top: 2%;
+  padding-left: 5%;
+  padding-bottom: 10%;
+  background-color: #f8f8f8;
+  border-radius: 0% 0% 90% 90%/0% 0% 100% 100%;
+}
+
+.student-registration-text {
+  padding: 2%;
+  font-weight: bold;
+  font-size: 20px;
+  line-height: 28px;
```

Comment:
Good work here, but consider using `rem` when styling your text, rather than `px`. This will avoid complications on the users end. Some users have very big screens and also zoom out their pages. Using `rem` help alot.

Reply:
done